pants

```diff
@@ -57,6 +57,7 @@ source ${HERE}/build-support/bin/native/bootstrap_code.sh
 
 # Default to using Python 3 if not otherwise specified.
 export PY="${PY:-python3}"
```

Comment:
Unrelated to this PR, we should adopt the idiom from the setup repo to first try `python3.6` then `python3.7` then `python3.8` etc. Will fix the issue of Pants devs having Python 3.5 installed on their computer and that being used by this script.